Packs/EDL/Integrations/EDL/EDL_description.md

```diff
@@ -17,21 +17,23 @@ In a web browser, go to **http://<cortex-xsoar-server-address>:<listen_port>**.
 <~XSIAM>
 **Note**: Do not set `username` and `password` in the integration instance if you are running the integration via the hosted instance. The `username    and `password` fields are for usage when running the integration via an on-prem engine.
 
+**Note**: If no `Listen Port` param was given and the test button was clicked, the test will run with the default port 1111. After pressing `save & exit` a new free port will be assigned to the `Listen Port` parameter automatically.
```

Comment:
This description is not accurate.
In XSOAR 6.x it'll still require to fill the port manually, or keep using 1111, no?

Reply:
This Note is in the <XSIAM></~XSIAM> so i guess this note is not relevant for XSOAR. But after a second thought, it is relevant to XSOAR NG so i will need this documents for XSOAR as well.